packages/frameworks/src/frameworks.ts

```diff
@@ -1561,6 +1561,64 @@ export const frameworks = [
     buildCommand: null,
     getOutputDirName: async () => 'public',
   },
+  {
+    name: 'React Snowpack',
+    slug: 'react-snowpack',
+    demo: 'https://react-snowpack-delta.vercel.app',
+    logo:
+      'https://raw.githubusercontent.com/vercel/vercel/master/packages/frameworks/logos/react.svg',
+    tagline:
+      'Unlike traditional React application tooling, Snowpack saves you from getting bogged down with complex bundler setups and configuration files.',
+    description: 'A React app, bootstrapped with create-snowpack-app',
```

Comment:
It seems like `create-snowpack-app` doesn't work by itself but it requires a template. Which template did you use? Do we need other framework detection entries for each template, for example Vue?